run/parsedb.go

```diff
@@ -1,25 +1,12 @@
 package run
 
 import (
-	"github.com/pkg/errors"
-
 	"gnorm.org/gnorm/database"
-	"gnorm.org/gnorm/database/drivers/mysql"
-	"gnorm.org/gnorm/database/drivers/postgres"
 	"gnorm.org/gnorm/environ"
 )
 
 func getDBInfo(env environ.Values, cfg *Config) (*database.Info, error) {
```

Comment:
This function doesn't really do anything now. We can just call cfg.Driver.Parse where we used to call this.   And then I think this PR will be perfect.  Thanks so much.